.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish Pipeline-ai 🐍 distributions 📦 to PyPI and TestPyPI
+
+on: push
+
+jobs:
+  build-and-publish:
+    name: Build and publish Pipeline-ai 🐍 distributions 📦 to PyPI and TestPyPI
+    runs-on: ubuntu-18.04
```

Comment:
Is there a particular reason to pick 18.04? If not I'd be tempted to use `ubuntu-latest` instead.

Reply:
true